phoneApp/user.js

```diff
@@ -0,0 +1,96 @@
+class User {
+	constructor(user) {
+		this.user = user;
+		this.options = [
+			{option: 'message', class: 'glyphicon-comment'},
+			{option: 'call', class: 'glyphicon-earphone'},
+			{option: 'video', class: 'glyphicon-facetime-video'},
+			{option: 'mail', class: 'glyphicon-envelope'}
+		];
+		this.additionalOptions = [
+			'Notes', 'Send message', 'Share contact', 'Add to favorites', 'Share my location', 'Block this caller'
```

Comment:
same about that content. it would never come from the server. The rule DRY no need here it's static.
It could have different styles for example or different logic and it would hard to reuse your solution